content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: SLSA level 2 compliance for Eclipse Temurin
+date: "2022-11-11T12:00:00+00:00"
+author: sxa
+description: Eclipse Temurin by Adoptium is compliant with level 2 of the SLSA secure development framework.
+tags:
+  - Temurin
+  - Security
+---
+
+# Introduction to SLSA
+
+[SLSA](https://slsa.dev) is a framework with individual levels to indicate
+increasing levels of security which software products can work towards. 
+Today, we are pleased to announce that the Eclipse Temurin build and
+distribution process is compliant with Level 2 of the SLSA specification.
+
+What does this mean in practice? The [SLSA v0.1 levels are defined at this
+link](http://slsa.dev/spec/v0.1/levels) and this means we are compliant with
+levels 1 and 2 of the specification.
+
+## SLSA Level 1
+
+Level 1 means that the build process is full scripted and automated and we
+are now producing Software Bill Of Material (SBOM) documents describing the
+items which went into producing each of our builds which are published
+alongside the downloads.  We have been compliant with level 1 for some time:
+The level 1 requirements are:
+
+- **Build - scripted build**: Our build steps are defined completely by the
+  Jenkins pipelines from the [ci-jenkins-pipelines](https://github.com/adoptium/ci-jenkins-pipelines) repository and the
+  underlying scripts run on the build machines from the
+  [temurin-build](https://github.com/adoptium/temurin-build)
+  repository - starting at
+  [make-adopt-build-farm.sh](https://github.com/adoptium/temurin-build/blob/master/build-farm/make-adopt-build-farm.sh)
+
+- **Provenance - available**: We create an SBOM in OWASP CycloneDX format
+  along with our releases which contains all of the information about how
+  the builds are produced, which should allow someone to rebuild if required
```

Comment:
Has this claim of rebuilding via SBOM details been tested thoroughly? Is it strictly necessary for meeting Level 1?

Reply:
No, which is why the wording is "should". I can remove it if you prefer.